checkov/version.py

```diff
@@ -1 +1 @@
-version = '2.0.762'
+version = '2.0.767'
```

Comment:
can you revert your change here, this is done automatically when we release a new version

Reply:
done